internal/wrappers/export-http.go

```diff
@@ -166,18 +166,26 @@ func (e *ExportHTTPWrapper) GetScaPackageCollectionExport(fileURL string) (*ScaP
 	if err != nil {
 		return nil, err
 	}
+
 	resp, err := SendHTTPRequestByFullURL(http.MethodGet, fileURL, http.NoBody, true, viper.GetUint(commonParams.ClientTimeoutKey), accessToken, true)
 	if err != nil {
 		return nil, err
 	}
 	defer resp.Body.Close()
 
+	body, err := io.ReadAll(resp.Body)
```

Comment:
merge with ben changes, remove from your code

Reply:
@OrShamirCM 
can you please check what is the status of the fix by SCA?

Reply:
@OrShamirCM what is the status of the fix by SCA?

Reply:
SCA done. Please remove the BOM code